src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,89 @@
-export const Pagination = () => {};
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage?: number;
```

Comment:
Consider making `perPage` a required prop if it is always expected to be provided by the parent component. Otherwise, ensure that the default value of 1 is appropriate for all use cases.